web3/main.py

```diff
@@ -187,6 +187,5 @@ def ens(self):
     def ens(self, new_ens):
         self._ens = new_ens
 
-    def getGasPrice(self, transaction_params=None):
-        # TODO: sanity checks?
+    def get_gas_price(self, transaction_params=None):
```

Comment:
Is gas price a concept that belongs on the global `Web3` class? I don't think it has anything to do with swarm or whisper or other protocols that Web3 interacts with, it seems like an `eth`-only concept, and belongs in that box.  (I'm really asking: not sure if gas applies to anything outside the core ethereum protocol).

I don't think we're perfectly consistent on that distinction, but would rather avoid making it less consistent than it currently is.

Reply:
I agree, I felt it was out of place as well but decided to go with it. Happy to move to `Web3.eth.` if you are too @pipermerriam

Reply:
My bad, happy to have it moved to `web3.eth` as well.